io/src/pcd_io.cpp

```diff
@@ -279,6 +284,7 @@ pcl::PCDReader::readHeader (std::istream &fs, pcl::PCLPointCloud2 &cloud,
       if (line_type.substr (0, 6) == "HEIGHT")
       {
         sstream >> cloud.height;
```

Comment:
Should we add the same check of sstream as on line 275-276?

Reply:
The absence of throw here is the same as the original behavior and will be caught later at around 364, it is more helpful to have it so that I will add it.
```
  if (cloud.width * cloud.height != nr_points)
```